wurst/war3map.j

```diff
@@ -589,7 +595,6 @@ function CreateNeutralPassiveBuildings takes nothing returns nothing
     set u=CreateUnit(p, 'n00N', 1152.0, - 896.0, 305.930)
     set u=CreateUnit(p, 'h01H', - 1344.0, 6528.0, 270.000)
     set u=CreateUnit(p, 'h01F', 3712.0, - 7104.0, 270.000)
-    set u=CreateUnit(p, 'h00A', 2432.0, - 11136.0, 270.000)

```

Comment:
What are the 2 units removed from start?

Reply:
Classic troll recipe, I replaced it through wurst to update the buildings recipe